index.html

```diff
@@ -125,7 +125,7 @@ <h1 class="info__title">Comodidades</h1>
       </a>
     </div>
     <div class="footer__disclaimer">
-      <span>Web created by </span><a class="footer__footnote" href="https://github.com/rtsc8" target="_blank">RTSC8<a />
+      <span>Web created by </span><a class="footer__footnote" href="https://github.com/rtsc8" target="_blank">RTSC8</head>a>
```

Comment:
You pushed an error here. Before pushing anything please check your code and check if everything looks correct on the browser.